test/extensions/filters/http/original_src/config_test.cc

```diff
@@ -0,0 +1,54 @@
+#include <numeric>
+
+#include "envoy/config/filter/http/original_src/v2alpha1/original_src.pb.h"
+#include "envoy/config/filter/http/original_src/v2alpha1/original_src.pb.validate.h"
+
+#include "extensions/filters/http/original_src/config.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace OriginalSrc {
+namespace {
+
+// In keeping with the class under test, it would have made sense to call this ConfigTest. However,
+// when running coverage tests, that conflicts with tests elsewhere in the codebase.
+class OriginalSrcHttpConfigTest : public testing::Test {
+public:
+  Config makeConfigFromProto(
```

Comment:
does the function get you anything? seems like just doing `Config config(config_proto)` inline instead would be just as clean?

Reply:
It's something I like to do in UT. I find that constructors often change their function signature. Given that they're used in every single test, and in most cases, the tests end up using the same boilerplate constructor code, it reduces the amount of test churn caused by changes to the code.

We could argue that we could just keep that behaviour in the constructor itself (i.e. keep the single-argument version), but if it's not used in production, I don't like to have it in the production code -- I'd rather abstract it in the UT. 

It may be premature to do it now, but I don't find that it harms readability too much. Either way, I can change it if you feel strongly about it. :)

Reply:
Sounds good, I don't feel strongly about it